x-pack/plugins/observability_solution/infra/public/components/asset_details/hooks/use_metrics_charts.ts

```diff
@@ -0,0 +1,155 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+import { findInventoryModel } from '@kbn/metrics-data-access-plugin/common';
+import useAsync from 'react-use/lib/useAsync';
+
+export const useFlyoutMetricsCharts = ({
```

Comment:
Can we improve the name of this hook? Or maybe I don't understand its purpose. The other hooks are assets (host, kubernetes...), right? What flyout means here?

Reply:
Certainly. I'll think of a better and more meaningful name for it.

Reply:
@cauemarcondes let me know if the new names are better to understand the purpose of the hooks.